understanding/focus-appearance.html

```diff
@@ -53,389 +53,316 @@
+                        <figcaption>図 6 合格: フォーカスのあるボタンの両側にあるインジケータの矩形形は、それぞれ幅 9px 、高さ 28px である。合計は 504px<sup>2</sup> で、かろうじて最小値 480px2<sup>2</sup> を満たしている。
                         </figcaption>
                         
                      </figure>
                      
                      
                      <div class="note">
-                        <p class="note-title marker">Note</p>
-                        <p>Prefer using focus indicator techniques that scale with both the width and height
-                           of the focused control. Otherwise, if controls change size across different variations
-                           of a page (for example, in a responsive design), the indicator might meet the area
-                           requirement in some variations but not others. For example, in the above figure, if
-                           the width of the two highlight rectangles did not scale as the button grew wider,
-                           it would stop meeting the minimum area requirement if the button needed to grow any
-                           wider to accomodate a longer button label.
+                        <p class="note-title marker">注記</p>
+                        <p>フォーカスされたコントロールの幅と高さの両方に応じて拡大縮小するフォーカスインジケーターのテクニックを使用することを好む。もしそうでなければ、ページのさまざまなバリエーション (レスポンシブデザインなど) でコントロールのサイズが変更される場合、インジケータは一部のバリエーションでは領域要件を満たすが、他のバリエーションでは満たさない可能性がある。例えば、上記の図で、もしボタンの幅が広くなるにつれて二つのハイライトの矩形の幅が拡大縮小されなかったら、より長いボタンラベルに対応するためにボタンの幅をさらに広げる必要がある場合に、最小面積要件を満たさなくなるだろう。
                         </p>
                      </div>
                      
                      
-                     <p>Another way of achieving the area requirement is to alter the appearance of the entire
-                        component, for instance by changing its color – provided that the new color has a
-                        contrast ratio of at least 3:1 against the original color. This can be effective in
-                        a set of closely placed buttons. The following example demonstrates this with 5 rating
-                        stars; the center star is filled in with a darker color to indicate focus. However,
-                        it is much more difficult to detect such a focus indicator when components are not
-                        near each other and so cannot be easily compared. For users using magnification, even
-                        components relatively close together may be difficult to compare, so it is not considered
-                        a best practice.
+                     <p>面積の要件を達成するもう一つの方法は、コンポーネント全体の外観を変更することである。例えば、色を変更して、新しい色が元の色に対するコントラスト比が少なくとも 3:1 にする方法がある。これは、近接して配置されたボタンのセットで効果的である。次の例は、五つの星の評価を使用してこれを示す。中央の星は、フォーカスを示すために濃い色で塗りつぶされている。ただし、コンポーネントが互いに近くになく、簡単に比較できない場合、このようなフォーカスインジケーターを検出することははるかに困難になる。拡大鏡を使用する利用者にとって、比較的近いコンポーネントであっても比較するのが難しい場合があるため、これはベストプラクティスとは見なされない。
```

Comment:
2番目のセンテンス (例えば...) を少し変えてみました。

```suggestion
                     <p>面積の要件を達成するもう一つの方法は、コンポーネント全体の外観を変更することである。例えば、色を変更して、新しい色が元の色に対して少なくとも 3:1 のコントラスト比を持つようにする方法がある。これは、近接して配置されたボタンのセットで効果的である。次の例は、五つの星の評価を使用してこれを示す。中央の星は、フォーカスを示すために濃い色で塗りつぶされている。ただし、コンポーネントが互いに近くになく、簡単に比較できない場合、このようなフォーカスインジケーターを検出することははるかに困難になる。拡大鏡を使用する利用者にとって、比較的近いコンポーネントであっても比較するのが難しい場合があるため、これはベストプラクティスとは見なされない。
```